.babelrc

```diff
@@ -1,20 +1,4 @@
 {
-  "presets": [
-    [
-      "es2015",
-      {
-        "modules": false
-      }
-    ],
-    "react"
-  ],
-  "plugins": ["transform-flow-strip-types"],
-  "env": {
-    "development": {
-      "presets": ["es2015", "react"]
-    },
-    "test": {
-      "presets": ["es2015", "react"]
-    }
-  }
+  "presets": ["@babel/env", "@babel/react"],
```

Comment:
Are we losing `modules: false` on purpose?

Reply:
Just confirmed that the files are built correctly, just in case.
According to rollup documentation, there is no need to set up any modules option:
https://github.com/rollup/rollup-plugin-babel#modules